.gitignore

Copyright (C) 2013 Curt Brune <curt@cumulusnetworks.com>

SPDX-License-Identifier:     GPL-2.0
---
 .gitignore |    1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index be1d461..08642ae 100644
--- a/.gitignore
+++ b/.gitignore
@@ -36,6 +36,7 @@ core
 .gdb_history
 .gdbinit
 
+busybox.links
 #
 # testing output
 #
